controllers/tweets.controller.js

```diff
@@ -1,5 +1,216 @@
-class TweetsController{
+import Tweet from "../models/tweets.model.js";
+import Like from "../models/likes.model.js";
+import Comment from "../models/comments.model.js";
+import User from "../models/users.model.js";
 
-}
+// Create a new tweet
+const createTweetController = async (req, res) => {
```

Comment:
Just create a class and write corresponding controller functions in it. Don't write Controller after each function. 

Always use class-based controllers rather than function-based based. It helps us remove the duplicity in functions during object creation.